core/web/router.go

```diff
@@ -279,6 +279,8 @@ func v2Routes(app chainlink.Application, r *gin.RouterGroup) {
 
 		txs := TransactionsController{app}
 		authv2.GET("/transactions/evm", paginatedRequest(txs.Index))
+		evmTxs := NewEVMTransactionController(app)
+		authv2.POST("/transactions/evm", auth.RequiresAdminRole(evmTxs.Create))
```

Comment:
Do we know what is the admin role here?
Who owns this, and how can we ensure that caller can be an admin?
Would we need to discuss this with EngOps?
I think EngOps could be the admins, and they may not want to delegate admin role to the Config Distribution client. We may perhaps need another role or something equivalent.

For now this code is ok.